pvanalytics/features/daytime.py

```diff
@@ -212,6 +219,13 @@ def power_or_irradiance(series, outliers=None,
     night = ((low_value & low_diff)
              | (low_value & low_median)
              | (low_diff & low_median))
+    # Nullify cases where the classification lasts 30 minutes or less
+    night_duplicates = _run_lengths(night)
+    if nullify_repeat_count is None:
+        nullify_repeat_count = int(30 / minutes_per_value)
+    night.loc[night_duplicates <= nullify_repeat_count] = np.nan
```

Comment:
Probably we should add test(s) to verify that this new parameter behaves as expected

Reply:
Removed so no longer needed